__tests__/app.test.js

```diff
@@ -55,13 +48,25 @@ describe("GET /api/topics", () => {
       });
   });
 });
-describe("GET /api/articles", () => {
+describe.only("GET /api/articles/:article_id", () => {
   test("200: responds with an array of articles", () => {
     return request(app)
-      .get("/api/articles")
+      .get("/api/articles/1")
       .expect(200)
-      .then(({ body: { articles } }) => {
-        expect(articles).toEqual(endpointsJson);
+      .then(({ body }) => {
+        console.log(body);
+        body.rows.forEach((articles) => {
+          expect(articles).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+          });
```

Comment:
There is no error handling in your test suite! In what ways could a user incorrectly interact with this endpoint?